src/FileSystemUtilities.js

```diff
@@ -65,6 +65,14 @@ export default class FileSystemUtilities {
 
   @logger.logifySync
   static unlinkSync(filePath) {
-    fs.unlink(filePath);
+    fs.unlinkSync(filePath);
+  }
+
+  @logger.logifySync
+  static isSymlink(path) {
+    const lstat = fs.lstatSync(path);
```

Comment:
`init` is the only command to use `unlinkSync`, and it checks for the file first. the confusing part is that it's calling `FileSystemUtilities.unlinkSync(versionLocation, "0.0.0");`.

not sure why that second argument is in there.

not sure this falls under this PR. i'll remove the change. it's out of scope and has no effect with these changes. i'll file a separate issue.